### ISSUE TITLE
Bug in Println line 96

### ISSUE THREAD

**Issue by davetorok** [aperepel/nifi-api-deploy#17]
sorry... too lazy to do a PR.

```
Undeploying Controller Service: StandardHttpContextMapCaught:   
groovy.lang.MissingPropertyException: No such property: println for class: NiFiDeploy 
groovy.lang.MissingPropertyException: No such property: println for class: NiFiDeploy
    at NiFiDeploy$_handleUndeploy_closure5.doCall(NiFiDeploy.groovy:96)
    at NiFiDeploy.handleUndeploy(NiFiDeploy.groovy:80)
    at NiFiDeploy.run(NiFiDeploy.groovy:554)
```

fixed with: 
`96:    println ""`

**Comment by aperepel** [comment]
Thank you Dave, I've had it fixed a while back, but sitting in my local git, not yet pushed :)

**Comment by aperepel** [comment]
Closed via 566bd9196191b78169d65ecc6c6b34ea83e52241
